lib/core/public/finish-run.js

```diff
@@ -0,0 +1,40 @@
+import { getReporter } from './reporter';
+import { mergeResults, publishMetaData, finalizeRuleResult, DqElement } from '../utils';
+
+export default function finishRun(data, options = {}) {
```

Comment:
It wasn't clear what `data` was in this function, had to read the tests to understand it was an array of `runPartial` results.